app/layout.tsx

```diff
@@ -0,0 +1,28 @@
+import '../components/globals.css';
+import { Manrope, Young_Serif } from 'next/font/google';
+import LayoutComponent from '../components/LayoutComponent';
+
+const manrope = Manrope({
+  display: 'swap',
+  weight: ['400', '700'],
+  subsets: ['latin'],
+  variable: '--font-manrope',
+});
+
+const youngSerif = Young_Serif({
+  weight: ['400'],
+  subsets: ['latin'],
+  variable: '--font-young-serif',
+});
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en" className={`${manrope.variable} ${youngSerif.variable}`}>
+      <LayoutComponent>{children}</LayoutComponent>
+    </html>
+  );
+}
```

Comment:
VickOnRails used `use client` inside the layout but we avoid making pages or layouts into Client Components. Instead, we create a new Client Component and use it in our layout. In this case, I created a `LayoutComponent` inside the components folder.